spec/support/fixture_builder.rb

```diff
@@ -121,6 +121,7 @@
 
     # Deploys
     name(:deploy, create(:deploy))
+    create(:deploy)
```

Comment:
We are adding a [spec](https://github.com/davidrunger/david_runger/pull/6018/files#diff-4399d8edfb259ed764255b47aa8ff01a2cba4810562b824124a69e57915ff451R5-R10) that makes sure we have at least two deploys. We might as well create them upfront here in the fixtures.